Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/FRFS/Init.hs

```diff
@@ -121,7 +121,7 @@ tfPayments booking mPaymentParams mSettlementType = do
   let mCurrency = Just booking.price.currency
   Just $
     singleton $
-      Utils.mkPayment Spec.NOT_PAID (Just $ encodeToText booking.price.amount) Nothing mPaymentParams mSettlementType mCurrency
+      Utils.mkPayment Spec.NOT_PAID (Just $ encodeToText booking.price.amount) Nothing mPaymentParams mSettlementType mCurrency (show <$> booking.delayedInterest) --handle here
```

Comment:
what needs to be handled here?

Reply:
missed removing comment its already handled `show <$> booking.delayedInterest)`